samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -1337,6 +1358,86 @@ int link_shell(const struct shell *shell, size_t argc, char **argv)
 		link_rsrp_subscribe(true) :
 		link_rsrp_subscribe(false);
 		break;
+	case LINK_CMD_SEARCH:
```

Comment:
No core business or code structure comments on this code block (entire `case` statement) yet. We need to resolve how to do the setting of parameters first and I'll see how this needs to be refactored into a function etc.

Reply:
This should be fine now. Requires some more testing tomorrow but ready for review. I need to add something to readme file too.